docs/spark-procedures.md

```diff
@@ -803,8 +820,11 @@ reports this as the following pair of rows, despite it not being an actual chang
 | 1   | Alice | DELETE       |
 | 1   | Alice | INSERT       |
 
-By default, this view finds the carry-over rows and removes them from the result. User can disable this
-behavior by setting the `remove_carryovers` option to `false`.
+By default, this view finds the carry-over rows and removes them from the result. Please query `SparkChangelogTable` if
```

Comment:
Suggestion:

``` To see carry-over rows, query `SparkChangelogTable`.```
(I think you mentioned already that carry-over rows are removed in the first sentence of the section.